lib/broccoli/ember-app.js

```diff
@@ -63,21 +63,9 @@ module.exports = EmberApp;
   @constructor
   @param {Object} options Configuration options
 */
-function EmberApp() {
-  var args = [];
-  var options = {};
-
-  for (var i = 0, l = arguments.length; i < l; i++) {
-    args.push(arguments[i]);
-  }
-
-  if (args.length === 1) {
-    options = args[0];
-  } else if (args.length > 1) {
-    args.reverse();
-    options = defaults.apply(null, args);
-  }
 
+function EmberApp() {
+  var options = this._extractOptions(arguments);
```

Comment:
Unfortunately merely passing arguments to another function will cause the issue.

Again, in this case it really doesn't matter, my brain is just hard wired to avoid those cases... It's hard for me to let go.

@rwjblue I am over thinking this right?